src/test/java/edu/ucsb/cs156/courses/controllers/PSCourseControllerTests.java

```diff
@@ -481,6 +515,8 @@ public void api_courses_post__user_logged_in__section_order_in_json_does_not_mat
 	PSCourse expectedSecondary = PSCourse.builder().enrollCd("08326").psId(1L).user(u).id(0L).build();
         when(coursesRepository.save(eq(expectedSecondary))).thenReturn(expectedSecondary);
 
+        when(coursesRepository.findByPsIdAndEnrollCd(eq(1L), eq("08326"))).thenReturn(Optional.empty());
```

Comment:
I think the enroll code should be "08292" since that's the primary enroll code and what gets checked in the `postCourses` implementation.